worldguard-core/src/main/java/com/sk89q/worldguard/protection/flags/Flags.java

```diff
@@ -134,6 +134,8 @@
     public static final LocationFlag TELE_LOC = register(new LocationFlag("teleport", RegionGroup.MEMBERS));
     public static final LocationFlag SPAWN_LOC = register(new LocationFlag("spawn", RegionGroup.MEMBERS));
 
+    public static final StringFlag TELE_MESSAGE = register(new StringFlag("teleport-message", "\u00A7dTeleported you to the region '%id%'."));
```

Comment:
Can you use the legacy serializer and text components as the other message flags do?

Reply:
Of course.